conjur/api/api.py

```diff
@@ -431,27 +432,29 @@ def list_members_of(self, uri_parameters: ListMembersOfData = None) -> dict:
 
         return json.loads(json_response.decode('utf-8'))
 
-    def list_permitted_members_of(self, uri_parameters: ListMembersOfData = None) -> dict:
+    def list_permitted_roles(self, data: ListPermittedRolesData) -> dict:
         """
         Lists the roles which have the named permission on a resource.
         """
+        if not data.kind:
+            raise MissingRequiredParameterException("Missing required parameter, 'kind'")
```

Comment:
@shulifink @szamir1 Please review text

Reply:
@szamir1  - not sure which one you're referring to.